test/browser/babel.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
Is Babel needed for Cypress specifically then?

Reply:
`cypress-react-unit-test` uses webpack to run/mount the test components, so babel is needed to process the js & ts files for webpack. `cypress-react-unit-test` supports several ways to run the code (webpack, just babel, next.js, etc) and webpack + babel was the closest to our normal flow (and I couldn't do just babel because there's some css used).

https://github.com/bahmutov/cypress-react-unit-test/blob/main/docs/recipes.md